python/CMakeLists.txt

```diff
@@ -29,12 +29,17 @@ target_include_directories(${SWIG_MODULE_xcfun_swig_REAL_NAME}
     ${PYTHON_INCLUDE_DIRS}
     ${NUMPY_INCLUDE_DIRS}
   )
+if(APPLE)
+  set(ORIGIN "@rpath/../../..")
```

Comment:
If you put `@loader_path` instead of `@rpath` here it works for me on MacOS.

Reply:
OK

Reply:
Seems to also fix the Travis failure! Yay!

Reply:
I'm trying one more thing, then cleaning up history a bit and this is finally ready to go.